server/resp/src/main/java/org/infinispan/server/resp/RespServer.java

```diff
@@ -90,7 +90,7 @@ protected void startInternal() {
             }
             builder.encoding().key().mediaType(RESP_KEY_MEDIA_TYPE);
             builder.encoding().value().mediaType(configuredValueType);
-            builder.statistics().enable();
+            builder.statistics().enable().aliases("0");
```

Comment:
Just seeking clarification, but will RESP explicitly ask for cache 0 ? And this should probably outside of this else so it always applied whenever an explicit configuration isn't supplied right?

Reply:
RESP doesn't explicitly ask for it. I'd also prefer not to enforce it in other cases

Reply:
Okay, that is what I figured, I still think we need to set the alias outside the else block so it is applied whenever an explicit configuration isn't supplied though.

Reply:
Ok